news_scraper/spiders/businesstoday.py

```diff
@@ -11,7 +11,7 @@ class BusinessTodaySpider(SitemapIndexSpider):
         "https://www.businesstoday.in/rssfeeds/date-wise-story-sitemap.xml?yyyy={year}&mm={month}&dd={day}",
     ]
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"/industry/", "parse"),(r"/india-at-100/", "parse")]
```

Comment:
`/india-at-100` doesnt seem to be a category, more like a general event and some articles under it.

Reply:
Yes its an event (economy summit) I saw few company names and news so I thought of including it.

Reply:
Okay, drop this for now. Way too specific.